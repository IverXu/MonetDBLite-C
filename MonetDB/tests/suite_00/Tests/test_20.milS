setoid(oid(20000000));
# test single column BATs
a:= new(int,void);
a.insert(1,void(nil));
a.print;
a.insert(2,void(nil));
a.insert(int(nil),void(nil));
a.print;

b:= new(void,int);
b.insert(void(nil),1);
b.print;
b.insert(void(nil),2);
b.insert(void(nil),int(nil));
b.print;
quit;
