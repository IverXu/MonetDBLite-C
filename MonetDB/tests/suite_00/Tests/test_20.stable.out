stdout of test 'test_20` in directory 'tests/suite_00` itself:

#> Mtest:	milS test_20 milS
#> Mtest:	Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -db tests_suite_00 -single test_20.milS
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.
#script:test_20.milS

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!
setoid(oid(20000000));
# test single column BATs
a:= new(int,void);
a.insert(1,void(nil));
a.print;
#-----------------#
# BAT:	  tmp_25  #
# (int)	  (void)  #
#-----------------#
[ 1,	  nil	  ]
a.insert(2,void(nil));
a.insert(int(nil),void(nil));
a.print;
#-----------------#
# BAT:	  tmp_25  #
# (int)	  (void)  #
#-----------------#
[ 1,	  nil	  ]
[ 2,	  nil	  ]
[ nil,	  nil	  ]

b:= new(void,int);
b.insert(void(nil),1);
b.print;
#-----------------#
# BAT:	  tmp_26  #
# (void)  (int)	  #
#-----------------#
[ nil,	  1	  ]
b.insert(void(nil),2);
b.insert(void(nil),int(nil));
b.print;
#-----------------#
# BAT:	  tmp_26  #
# (void)  (int)	  #
#-----------------#
[ nil,	  1	  ]
[ nil,	  2	  ]
[ nil,	  nil	  ]
quit;
